lib/tasks/index.rake

```diff
@@ -0,0 +1,61 @@
+namespace :hyku do
+  desc "reindex just the works in the background"
+  task reindex_works: :environment do
+    Account.find_each do |account|
+      puts "=============== #{account.name}============"
+      next if account.name == "search"
+      switch!(account)
+      begin
+        Site.instance.available_works.each do |work_type|
+          title = "~ #{account.name} - #{work_type}"
+          klass = work_type.constantize
+          progressbar = ProgressBar.create(total: klass.count, title: title, format: "%t %c of %C %a %B %p%%")
```

Comment:
In Adventist, I was able to run this task locally but wasn't able to in prod without requiring progressbar at the top of this file. 

ref: https://github.com/scientist-softserv/adventist-dl/pull/465/files

Reply:
thank you i will look at this!